src/test/groovy/graphql/DataFetcherTest.groovy

```diff
@@ -98,7 +98,7 @@ class DataFetcherTest extends Specification {
         given:
         def environment = env(GraphQLString)
         when:
-        def result = new PropertyDataFetcher("publicField").get(environment)
+        def result = new FieldDataFetcher("publicField").get(environment)
         then:
         result == "publicValue"
     }
```

Comment:
The point is that `Boolean isBooleanMethod` should not work, because by convention you either have `boolean isBooleanMethod` or `Boolean getBooleanMethod`. These two cases were covered in the existing testcase: https://github.com/athenagroup/graphql-java/blob/fa6eefae0fcecb6bfc991d275884bd7d0a065cf3/src/test/groovy/graphql/DataFetcherTest.groovy#L102-L118